src/components/Hero/hero.style.js

```diff
@@ -71,58 +70,61 @@ H.ImgContainer = styled.div`
     bottom: -1px;
     background: #202125;
     background: -moz-linear-gradient(
-      0deg,
-      #202125 0,
-      rgba(32, 33, 37, 0) 50%,
-      #202125 100%
+            0deg,
+            #202125 0,
+            rgba(32, 33, 37, 0) 50%,
+            #202125 100%
     );
     background: -webkit-linear-gradient(
-      0deg,
-      #202125 0,
-      rgba(32, 33, 37, 0) 50%,
-      #202125 100%
+            0deg,
+            #202125 0,
+            rgba(32, 33, 37, 0) 50%,
+            #202125 100%
     );
     background: linear-gradient(
-      0deg,
-      #202125 0,
-      rgba(32, 33, 37, 0) 50%,
-      #202125 100%
+            0deg,
+            #202125 0,
+            rgba(32, 33, 37, 0) 50%,
+            #202125 100%
     );
   }
   @media screen and (max-width: 759px) {
     &::after {
       background: #202125;
       background: -moz-linear-gradient(
-        0deg,
-        #202125 0,
-        rgba(32, 33, 37, 0) 82%
+              0deg,
+              #202125 0,
+              rgba(32, 33, 37, 0) 82%
       );
       background: -webkit-linear-gradient(
-        0deg,
-        #202125 0,
-        rgba(32, 33, 37, 0) 82%
+              0deg,
+              #202125 0,
+              rgba(32, 33, 37, 0) 82%
       );
       background: linear-gradient(0deg, #202125 0, rgba(32, 33, 37, 0) 82%);
     }
     &::before {
-      content: '';
+      content: "";
       opacity: 0;
     }
   }
-`
+`;
 
 H.Img = styled.img`
-  width: 100%;
+  width: 70%;
```

Comment:
img needs to be 100% also without left 30%

Reply:
i added that because the images were stretching a lot and it was compromising the resolution